src/current/v24.1/known-limitations.md

```diff
@@ -6,699 +6,624 @@ keywords: gin, gin index, gin indexes, inverted index, inverted indexes, acceler
 
-### `AS OF SYSTEM TIME` does not support placeholders
+It is currently not possible to [add a column]({% link {{ page.version.version }}/alter-table.md %}#add-column) to a table when the column uses a [sequence]({% link {{ page.version.version }}/create-sequence.md %}) as the [`DEFAULT`]({% link {{ page.version.version }}/default-value.md %}) value, for example:
 
-CockroachDB does not support placeholders in [`AS OF SYSTEM TIME`]({% link {{ page.version.version }}/as-of-system-time.md %}). The time value must be embedded in the SQL string.
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> CREATE TABLE t (x INT);
+~~~
 
-[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/30955)
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> INSERT INTO t(x) VALUES (1), (2), (3);
+~~~
 
-### CockroachDB does not properly optimize some left and anti joins with GIN indexes
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> CREATE SEQUENCE s;
+~~~
 
-[Left joins]({% link {{ page.version.version }}/joins.md %}#left-outer-joins) and anti joins involving [`JSONB`]({% link {{ page.version.version }}/jsonb.md %}), [`ARRAY`]({% link {{ page.version.version }}/array.md %}), or [spatial-typed]({% link {{ page.version.version }}/export-spatial-data.md %}) columns with a multi-column or [partitioned]({% link {{ page.version.version }}/alter-index.md %}#partition-by) [GIN index](inverted-indexes.html) will not take advantage of the index if the prefix columns of the index are unconstrained, or if they are constrained to multiple, constant values.
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> ALTER TABLE t ADD COLUMN y INT DEFAULT nextval('s');
+~~~
 
-To work around this limitation, make sure that the prefix columns of the index are either constrained to single constant values, or are part of an equality condition with an input column (e.g., `col1 = col2`, where `col1` is a prefix column and `col2` is an input column).
+~~~
+ERROR: nextval(): unimplemented: cannot evaluate scalar expressions containing sequence operations in this context
+SQLSTATE: 0A000
+~~~
 
-For example, suppose you have the following [multi-region database]({% link {{ page.version.version }}/multiregion-overview.md %}) and tables:
+[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/42508)
 
-``` sql
-CREATE DATABASE multi_region_test_db PRIMARY REGION "europe-west1" REGIONS "us-west1", "us-east1" SURVIVE REGION FAILURE;
-USE multi_region_test_db;
+#### Dropping a column referenced by a partial index
 
-CREATE TABLE t1 (
-  k INT PRIMARY KEY,
-  geom GEOMETRY
-);
+{% include {{ page.version.version }}/known-limitations/drop-column-partial-index.md %}
 
-CREATE TABLE t2 (
-  k INT PRIMARY KEY,
-  geom GEOMETRY,
-  INVERTED INDEX geom_idx (geom)
-) LOCALITY REGIONAL BY ROW;
-```
+#### Schema change DDL statements inside a multi-statement transaction can fail while other statements succeed
```

Comment:
Some of these headings are sentences, like this one, and others are fragments, like the next one. Is this intentional?

Reply:
The ones written like sentences are existing headings. I don't have the bandwidth to edit all of the headings, and this PR is more about reorganizing the given content. I did rename/remove headings opportunistically.

Reply:
SGTM